qiskit/circuit/tools/pi_check.py

```diff
@@ -155,7 +155,7 @@ def normalize(single_inpt):
             if denom == 1 and output != "qasm":
                 denom = ""
             if output == "latex":
-                str_out = "\\frac{%s%s}{%s%s}" % (neg_str, numer, denom, pi)
+                str_out = f"\\frac{{{neg_str}{numer}}}{{{denom}{pi}}}"
```

Comment:
This (and similar above) _may_ be cases where the old style % formatting is clearer.  I'm not particularly bothered, though - I generally think f-strings are better in most cases.

Reply:
I could be persuaded either way. Left it with f-strings for now

Reply:
Yeah, this one's got enough components to it, and few enough nested "{}" pairs that the f-string is probably better.